src/main/java/com/cognifide/knotx/service/MockRemoteRepositoryHandler.java

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
It would be good to separate this service from Knot.x in the future (e.g. as separate Vert.x verticle) since it is just for repository mocking.